### PR TITLE
Fix space in paths

### LINKED ISSUE #8005: Cannot process files
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | 0.17.1          |

Tried to run the `./vendor/bin/rector process ` and got **fatal errors file /media/mac/New does not exist. Could not process some files, due to:                                                                          
         "Child process error: ".** 

I am using Ubuntu 22

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes https://github.com/rectorphp/rector/issues/8005

### CODE REVIEW

File: src/Configuration/ConfigurationFactory.php [hunk 1]
```diff
@@ -93,32 +93,16 @@ private function shouldShowDiffs(InputInterface $input): bool
...
-    private function correctBashSpacePaths(array $commandLinePaths): array
-    {
-        // fixes bash edge-case that to merges string with space to one
-        foreach ($commandLinePaths as $commandLinePath) {
-            if (\str_contains($commandLinePath, ' ')) {
-                $commandLinePaths = explode(' ', $commandLinePath);
-            }
-        }
-
-        return $commandLinePaths;
```
Comment (TomasVotruba) [inline src/Configuration/ConfigurationFactory.php:109]: This actually incorrectly splits any spaced path to 2 strings:

```bash
/some directory/file.php
```

will result into:

```bash
/some
directory/file.php
```

**Comment by samsonasik** [comment]
@paulbalandan could you verify this on windows? Try latest rector/rector:dev-main. Thank you.

**Comment by paulbalandan** [comment]
@samsonasik I used `rector/rector dev-main ad6f93a`
```pwsh
PS C:\Users\P\Desktop\Web Dev\CodeIgniter4> vendor/bin/rector process system
Could not open input file: C:\Users\P\Desktop\Web
```

**Comment by samsonasik** [comment]
@paulbalandan thank you for check.

@TomasVotruba it may need to use e2e for windows.

@paulbalandan could you help create a reproducible e2e or github repo with windows running ( vendor committed ) to allow debug in there? Thank you.

**Comment by samsonasik** [comment]
@TomasVotruba the issue on windows seems the root path before project path that may contains space

```
:\Users\P\Desktop\Web Dev\CodeIgniter4> vendor/bin/rector

**Comment by TomasVotruba** [comment]
@samsonasik I see :+1: . I don't use Windows so I can't help much. Having a e2e test case would give us at least regression check.

**Pull request merged**